checkstyle.xml

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0"?>
```

Comment:
lets replace this file by appropriate updates of pom.xml (u can find them in ur previous projects